packages/host/app/controllers/card.ts

```diff
@@ -7,18 +7,26 @@ import { action } from '@ember/object';
 
 import { tracked } from '@glimmer/tracking';
 const { isLocalRealm } = ENV;
+import { ComponentLike } from '@glint/template';
+import { Model } from '@cardstack/host/routes/card';
 
 export default class CardController extends Controller {
   isLocalRealm = isLocalRealm;
-  model: any;
+  isolatedCardComponent: ComponentLike | undefined;
   withPreventDefault = withPreventDefault;
   @service declare router: RouterService;
   @tracked operatorModeEnabled = false;
+  @tracked model: Model | undefined;
```

Comment:
Without adding `@tracked`, the template didn't rerender on model change. I thought the model is tracked by default 🤔

Reply:
interesting, i thought the same thing as well...